adafruit_wiznet5k/adafruit_wiznet5k.py

```diff
@@ -175,6 +175,14 @@ def __init__(
         self.mac_address = mac
         self._src_port = 0
         self._dns = 0
+
+        # First wait link status is on
+        # to avoid the code during DHCP - assert self.link_status, "Ethernet cable disconnected!"
+        while True:
+            time.sleep(1)
+            print("My Link is:", self.link_status)
```

Comment:
The print statement should only print in debug mode like the other prints.

```suggestion
            if self._debug:
                        print("My Link is:", self.link_status)
```